mne/beamformer/_lcmv.py

```diff
@@ -437,7 +441,7 @@ def _lcmv_source_power(info, forward, noise_cov, data_cov, reg=0.05,
     # Tikhonov regularization using reg parameter to control for
     # trade-off between spatial resolution and noise sensitivity
     # This modifies Cm inplace, regularizing it
-    Cm_inv, d = _reg_pinv(Cm, reg)
+    Cm_inv, d, _ = reg_pinv(Cm, reg)
```

Comment:
Also let's keep `reg_pinv` private for now (I can do this, too, if I'm going to update the docstrings)

Reply:
Good idea. Feel free to do this yourself :)